spec/requests/api/orchestration_template_spec.rb

```diff
@@ -125,4 +125,70 @@
       expect(OrchestrationTemplate.exists?(hot.id)).to be_falsey
     end
   end
+
+  context 'orchestration template copy' do
+    it 'copy template with the same content' do
+      test_ot_1 = FactoryGirl.create(:orchestration_template_cfn)
+
+      api_basic_authorize collection_action_identifier(:orchestration_templates, :copy)
+
+      run_post(orchestration_templates_url(test_ot_1.id), gen_request(:copy))
+      expect(response).to have_http_status(:bad_request)
+    end
+
+    it 'copy single template with the different content' do
+      test_ot_1 = FactoryGirl.create(:orchestration_template_cfn)
+      test_content_1 = "{ 'Description': 'Test content 1' }\n"
```

Comment:
I think it because we add new line in the orchestration template by intention(I can see it from code)
`text.encode(:universal_newline => true).chomp.concat("\n")`